app/controllers/posts_controller.rb

```diff
@@ -2,26 +2,35 @@ class PostsController < ApplicationController
   # TODO (Shimmy): Define logged_in_user
   # before_action :logged_in_user, only: [:create, :destroy]
 
+  def index
+    @group = Group.friendly.find(params[:group_id])
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  # TODO (Shimmy): Watch out for duplicates
+  # TODO (Shimmy): Flash success notification on send
   def create
     @group = Group.friendly.find(params[:group_id])
     @post = @group.posts.build(post_params)
-    @post.user_id = current_user.id
-    if @post.save
-      redirect_to group_path(@group), notice: "Post has been saved successfully."
-    end
+    @post.save
```

Comment:
@negativetwelve 

`post_params` doesn't contain a group_id. I thought adding `@post = Post.new(group_id: @group.id)` in the GroupsController would fix it, but even then `post_params` has a nil `group_id`. Seems like the only easy way around this is to specify the `group_id` in this create method..